frontend/src/pages/Traces/TraceSpanAttributes.tsx

```diff
@@ -1,14 +1,74 @@
 import JsonViewer from '@/components/JsonViewer/JsonViewer'
 import { FlameGraphSpan } from '@/pages/Traces/utils'
 
-type Props = { span: FlameGraphSpan }
+type Props = {
+	span: FlameGraphSpan
+}
 
 export const TraceSpanAttributes: React.FC<Props> = ({ span }) => {
-	const attributes = { ...span }
+	let attributes: { [key: string]: any } = { ...span }
 
 	// Drop any attributes we don't want to display
 	delete attributes.__typename
 	delete attributes.children
+	delete (attributes as any).projectID
+
+	// Convert timestamp to a formatted date/time
+	if (attributes.timestamp) {
+		attributes.timestamp = new Date(attributes.timestamp).toLocaleString()
+	}
+
```

Comment:
could we also format the `duration` attribute nicely? currently it's just the nanosecond value, but maybe we could label that or convert to seconds and label it `s`?

Reply:
Handled in https://github.com/highlight/highlight/pull/7517/commits/281149eb92333157202c20762c905e3fc414d780.